db/seed/synthetic_studies/human_raw_counts/study_info.json

```diff
@@ -0,0 +1,29 @@
+{
+  "study": {
+    "name": "raw_1",
```

Comment:
Update the study name here -- "Human raw counts"

Reply:
Thanks for catching that! fixed `7be9c110`